docs/docs/keymaps/input-processors/behaviors.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Behaviors Input Processor
+sidebar_label: Behaviors
+---
+
+## Overview
+
+The behaviors input processor is used invoke standard behaviors when certain input events occur; most frequently this is used to trigger behaviors when certain mouse buttons are triggered by physical pointing devices. For split devices, this is used with listeners on the central, and will never invoke behaviors on the split peripheral devices.
+
+:::note
+
+The behaviors input processor is primarily intended for `INPUT_EV_KEY` type of events that have a binary on/off state, not vector types for relative or absolute movements.
+
+:::
+
+## Usage
+
+When used, this input processor takes no parameters, as the event code to behavior mapping is specified in the definition of the specific processor instance, e.g.:
+
+```dts
+&zip_button_behaviors
+```
+
+## Pre-Defined Instances
+
+One pre-defined instance of the out-of-band behaviors input processor is available:
+
+| Reference               | Description                                        |
+| ----------------------- | -------------------------------------------------- |
+| `&zip_button_behaviors` | Maps left/right/middle clicks to a given behavior. |
+
+Should you wish to update the existing instance to trigger different behaviors for each mouse button, you can override the `bindings` property, e.g.:
+
+```dts
+&zip_button_behaviors {
+    bindings = <&kp A &kp B &kp C>;
+};
+```
+
+By default, the `bindings` property maps all the buttons to [`&none`](../behaviors/misc.md#none).
```

Comment:
```suggestion
By default, the `bindings` property maps all the buttons to [`&none`](../behaviors/misc.md#none), so you will want to override the `bindings` property like above if you use this processor by assigning it to an [input listener](usage.md).
```

This might be a bit clearer.